src/codeu/chat/server/Model.java

```diff
@@ -680,4 +713,67 @@ public Uuid conversationID(Uuid message) {
     String query = "SELECT CONVERSATIONID FROM MESSAGES WHERE ID = ?;";
     return dbConnection.getConversationID(parameters, query);
   }
+
+  public void add(Uuid user) {
```

Comment:
Rename this to addCluster() or something similar? Also the parameter name should be something like cluserId I think if I'm understanding the code correctly.